codedigger/atcoder/tests/test_api.py

```diff
@@ -0,0 +1,21 @@
+from django import test
+from .test_setup import TestSetUp
+from ..api import *
+
+
+class Test(TestSetUp):
+    def test_atcoder_get_all_contests(self):
+        res = get_all_contests()
+        self.assertEqual(res[0]["id"], "APG4b")
```

Comment:
Make sure these data doesn't change with time.

Reply:
Yes, the API always returns this.